src/tribler/test_unit/core/restapi/test_ipv8_endpoint.py

```diff
@@ -12,8 +14,9 @@ def test_binding(self) -> None:
         """
         Test if all the IPv8 endpoint paths exist.
         """
-        endpoint = IPv8RootEndpoint()
-        endpoint.setup_routes()
+        with patch('asyncio.get_running_loop', lambda: None):
```

Comment:
This shows an IPv8 regression. The `ipv8/REST/tunnel_endpoint.py` should be changed: there is no reason to store the event loop in the `__init__`.